src/net_processing.cpp

```diff
@@ -1925,6 +1925,29 @@ bool static ProcessMessage(CNode* pfrom, std::string strCommand, CDataStream& vR
     return true;
 }
 
+static bool DisconnectIfBanned(CNode* pnode, CConnman* connman)
```

Comment:
Not a big deal but `SendRejectsAndCheckIfBanned` was renamed to `CheckIfBanned` in upstream after the removal of BIP61 (not to `DisconnectIfBanned`).
It would be good to keep the same name to make future backports easier (this is going to be renamed again with the introduction of the discouragement filter).

Reply:
I modified it because `CheckIfBanned` was not descriptive to what the function is really doing. The function is disconnecting the peer if banned, not just checking if it is banned.

but.. don't have a strong preference here. it was just a bad name that i corrected while was doing this changes. If upstream changed it again later, can just move it to the other provisory name.